### PR TITLE
Updated Homepage & Nav

### PR DESCRIPTION
**Pull request by kevinli-work**
* Redesigned navigation bar
* Updated homepage design
* Added link to analytics site to footer

### CODE REVIEW

File: src/js/components/sharedComponents/Footer.jsx [hunk 1]
```diff
@@ -71,6 +71,11 @@ export default class Footer extends React.Component {
                                             link="http://fedspendingtransparency.github.io/DAIMS-v1.1/"
                                             title="Data Model" />
                                     </li>
+                                    <li>
+                                        <FooterExternalLink
+                                            link="https://fedspendingtransparency.github.io/data-lab/"
```
Comment (ebdabbs) [inline src/js/components/sharedComponents/Footer.jsx]: I'm seeing a 404 when I follow this link

File: src/js/components/sharedComponents/header/NavBar.jsx [hunk 2]
```diff
@@ -1,72 +1,90 @@
...
-                            </a>
-                        </li>
-                        <li className="menu-item">
-                            <SearchBar />
-                        </li>
-                    </ul>
+            </div>
+            <div className="mobile-menu">
+                <div className="mobile-button-wrapper">
+                    <button className={`usa-menu-btn ${homepageClass}`}>
```
Comment (ebdabbs) [inline src/js/components/sharedComponents/header/NavBar.jsx]: Not expanding the menu on click

File: src/js/components/homepage/Homepage.jsx [hunk 3]
```diff
@@ -153,11 +154,27 @@ export default class Homepage extends React.Component {
         });
     }
 
+    skippedNav(e) {
+        // don't update the URL due to potential React Router conflicts
+        e.preventDefault();
+        // scroll to the main-content id
+        const yPos = document.querySelector('#main-content').getBoundingClientRect().top;
```
Comment (ebdabbs) [inline src/js/components/homepage/Homepage.jsx]: Pressing enter or clicking this button to skip the navigation scrolls the page but doesn't update the focus, so keyboard users still have to tab through the menu

File: src/js/components/sharedComponents/Footer.jsx (hunk 1 above)
Comment (kevinli-work) [reply src/js/components/sharedComponents/Footer.jsx]: Checked earlier today that this is the correct URL, supposedly this site will be ready when we deploy. If it's not ready by then, I'll remove.

File: src/js/components/sharedComponents/header/NavBar.jsx (hunk 2 above)
Comment (kevinli-work) [reply src/js/components/sharedComponents/header/NavBar.jsx]: Menu does not exist, in design currently. Will be its own story

File: src/js/components/sharedComponents/Footer.jsx (hunk 1 above)
Comment (ebdabbs) [reply src/js/components/sharedComponents/Footer.jsx]: Not deployed yet

File: src/js/components/sharedComponents/header/NavBar.jsx (hunk 2 above)
Comment (ebdabbs) [reply src/js/components/sharedComponents/header/NavBar.jsx]: Implementation will be in a future sprint

**Comment by AGuyNamedMarco** [comment]
## **Homepage Nav:**

screen shot 2017-09-07 at 4 25 08 pm

### Main (Center) Links
• **All**: Increase font-weight to "semibold" (600)
• **Inactive**: decrease opacity to 60%
• **Active (On Hover)**: keep opacity at 100%

### Dropdown Links
• **All**: decrease font-weight to "light" (200)
• **Disabled**: decrease opacity to 50%
• **Inactive**: decrease opacity to 60%
• **Active (On Hover)**: keep opacity at 100%

### Secondary (Right) Links
• **Inactive**: Decrease opacity to 60%
• **Active (On Hover)**: keep opacity at 100%


## **Non-Homepage Nav:**

screen shot 2017-09-07 at 4 24 52 pm

### Main (Center) Links
• **All**: Set default font-color to #555555
• **All**: Increase font-weight to "semibold" (600)
• **Inactive**: decrease opacity to 60%
• **Active (On Hover)**: keep opacity at 100%

### Dropdown Links
• **All**: Set default font-color to #555555
• **All**: decrease font-weight to "light" (200)
• **Disabled**: decrease opacity to 50%
• **Inactive**: decrease opacity to 60%
• **Active (On Hover)**: keep opacity at 100%

### Secondary (Right) Links
• **All**: Set default font-color to #555555
• **Inactive**: Decrease opacity to 60%
• **Active (On Hover)**: keep opacity at 100%

### Dropdown Menu
• Set background color to #F1F1F1
• Add drop-shadow to background (offset-x: 0, offset-y: 2, blur: 5, spread: 0)

**Comment by AGuyNamedMarco** [comment]
Look good! :thumbsup: Thanks!

**Pull request merged**